packages/web/curl/package.mk

```diff
@@ -53,6 +53,8 @@ PKG_CONFIGURE_OPTS_TARGET="ac_cv_lib_rtmp_RTMP_Init=yes \
                            --without-spnego \
                            --without-gssapi \
                            --with-zlib \
+                           --without-brotli \
+                           --without-zstd \
```

Comment:
can you compare the size impact if we enable zstd ?
from curl changelog
```
This is the first curl release that supports zstd compression. zstd is a yet another way to compressed content data over HTTP and if curl supports it, it can then automatically decompress it on the fly.
```

Reply:
Adding libzstd increased image 228KB for me. Commits are added here doing that, and support to curl. Curl reports: 
`accept-encoding: deflate, gzip, zstd` but I have no idea what websites may be using it.

Reply:
urgs, yea then  just keep it disabled for now

Reply:
Disabled for curl again. Moved the zstd configure options to the host build. Only gcc and squashfs-tools are using it, and they just need the lib so this should save a little time building.